lib/AST/Decl.cpp

```diff
@@ -420,7 +420,7 @@ bool Decl::isPrivateStdlibDecl(bool whitelistProtocols) const {
   if (auto AFD = dyn_cast<AbstractFunctionDecl>(D)) {
     // Hide '~>' functions (but show the operator, because it defines
     // precedence).
-    if (AFD->getNameStr() == "~>")
```

Comment:
Aha, yes, that's gone now that we have protocol extensions. Still, that should probably be a separate PR.